app/views/partials/multi_factor_authentication/_selected_mfa_option.html.erb

```diff
@@ -0,0 +1,8 @@
+<li class="usa-icon-list__item">
+    <div class="usa-icon-list__icon text-green">
+        <%= render IconComponent.new(icon: :check_circle, width: 24, height: 24) %>
```

Comment:
That's pretty neat how the icon inherits the text color! Maybe in the future it could be worth exploring making this an option of the component itself, so developers wouldn't need to concern themselves with thinking about text coloring and how the color is inherited with `currentColor`.

Something like...

```suggestion
    <div class="usa-icon-list__icon">
        <%= render IconComponent.new(icon: :check_circle, color: :green, width: 24, height: 24) %>
```